packages/core/src/oidc/init.ts

```diff
@@ -84,19 +84,20 @@ export default async function initOidc(app: Koa): Promise<Provider> {
     },
     // https://github.com/panva/node-oidc-provider/blob/main/recipes/client_based_origins.md
     clientBasedCORS: (_, origin, client) => isOriginAllowed(origin, client.metadata()),
+    claims: {
+      openid: ['sub', 'name', 'avatar', 'custom_data'],
+    },
     findAccount: async (ctx, sub) => {
-      await findUserById(sub);
+      const { name, avatar, customData } = await findUserById(sub);
```

Comment:
nit: maybe we can declare a custom Claims type

Reply:
The claims will change according to the scope in the future.
Claims Type makes no sense here.